app/controllers/admin/bulk_uploads_controller.rb

```diff
@@ -23,6 +23,9 @@ def upload_zip
   def create
     @bulk_upload = BulkUpload.new(@edition)
     @bulk_upload.attachments_attributes = create_params[:attachments_attributes]
+    @bulk_upload.attachments.each do |attachment|
+      attachment.attachment_data.attachable = @edition
+    end
```

Comment:
Minor: Did you consider adding a controller test for this behaviour? Don't worry if it's hard, but it might be good if you can.

Reply:
This was an oversight. I'll add a controller test.